bounties_api/std_bounties/models.py

```diff
@@ -225,6 +225,7 @@ class Fulfillment(models.Model):
     fulfiller_email = models.CharField(max_length=128, blank=True)
     fulfiller_githubUsername = models.CharField(max_length=128, blank=True)
     fulfiller_address = models.CharField(max_length=128, blank=True)
+    fulfiller_profile_image = models.CharField(max_length=256, blank=True)
```

Comment:
@c-o-l-o-r  why are you adding fulfiller_profile_image to the model? This isn't the right way to go about doing this. Lets talk :)

Reply:
There's a user relation... :) Just include it in the serializer.